core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -291,8 +292,12 @@ object SparkSubmit {
       } else {
         Nil
       }
+
+    val ivySettings = Option(args.ivySettingsFile).map(SparkSubmitUtils.loadIvySettings).getOrElse(
```

Comment:
If it's not much work I think it would be worth it to support the use case. If there's something that blocks that from working well, then it's ok to not support it.